src/graphql/mod.rs

```diff
@@ -48,7 +48,7 @@ pub async fn query_graph_node_poi(
         match data.proof_of_indexing {
             Some(poi) => Ok(poi),
             // _ => Err(QueryError::EmptyResponseError),
```

Comment:
😂I might've accidentally committed this, but this line definitely should not have been commented. Could you please uncomment the EmptyResponseError and then comment or delete the workaround on line#51?

Reply:
done ✅